backend/src/config/passportConfig.ts

```diff
@@ -92,3 +92,21 @@ passport.use('refresh-jwt', new CustomStrategy(async (req, done) => {
     return done(err);
   }
 }));
+
+const validateUser = async (id: string, done: VerifiedCallback) => {
+  try {
+    const user = await getCustomRepository(UserRepository).getById(id);
+    return user ? done(null, user) : done({ status: 401, message: 'Token is invalid.' }, null);
+  } catch (err) {
+    return done(err);
+  }
+};
+
+const options2 = {
```

Comment:
The object's name should be more informative, now it's not clear what is the difference between `options` and `options2`

Reply:
Solved